extensions/amp-analytics/0.1/vendors.js

```diff
@@ -552,6 +554,85 @@ export const ANALYTICS_CONFIG = /** @type {!JSONType} */ ({
     'optout': '_gaUserPrefs.ioo',
   },
 
+  // 'googleanalytics-legacycookie' is an exact copy of 'googleanalytics'
+  // except that it uses a different cookie name for CLIENT_ID
+  // It's introduced as a temporary solution for #5761.
+  'googleanalytics-legacycookie': {
```

Comment:
I think we should fix adsense and doubleclick then to read either.

This config makes much more sense long term.